### PR TITLE
Dropdown uses popper internally

### PR DESCRIPTION
**Pull request by l3satwik**

### CODE REVIEW

File: src/components/DropDown.tsx [hunk 1]
```diff
@@ -1,72 +1,55 @@
...
-              className={cx(dropDownStyle, dropDownClassName)}
-              style={{ padding }}
-            >
-              {children({ toggle: this.toggleDropdown })}
-            </div>
-          )}
-        </div>
-      </OutsideClick>
+        {({ toggle }) => (
+          <div className={cx(dropDownClassName)} style={{ padding }}>
```
Comment (azizhk) [inline src/components/DropDown.tsx]: why is padding given as style? move it to dropDownStyle

File: src/components/DropDown.tsx [hunk 2]
```diff
@@ -1,72 +1,55 @@
...
+                isSelected={isSelected}
+                isOpen={isOpen}
+                onClick={toggle}
+                disabled={disabled}
+                className={labelClassName}
+              >
+                {buttonLabel}
+              </DropDownButton>
+            )}
+          </React.Fragment>
```
Comment (azizhk) [inline src/components/DropDown.tsx]: why wrapped in React.Fragment?

File: src/components/DropDown.tsx [hunk 3]
```diff
@@ -1,72 +1,55 @@
...
-          })
-        }
-        disabled={!isOpen}
+      <Popper
+        isOpen={initiallyOpen}
+        placement="bottom"
+        label={({ isOpen, toggle }) => (
+          <React.Fragment>
+            {labelComponent ? (
+              labelComponent({ isOpen, toggleDropdown: toggle })
```
Comment (azizhk) [inline src/components/DropDown.tsx]: add stories with custom labelComponent as well

**Comment by azizhk** [comment]
Also update tests snapshot. @ritz078 will look at the diff in the snapshot to see if correct or not.

File: src/components/DropDown.tsx (hunk 2 above)
Comment (l3satwik) [reply src/components/DropDown.tsx]: Popper's label has return type defined as `JSX.Element`

Comment (ritz078) [reply src/components/DropDown.tsx]: Then change the typing. 😞

**Comment by l3satwik** [comment]
Closing this PR as it is been done in another PR by @ChaitanyaGadodia

**Pull request closed without merging**